models/_sector/dex/trades/zksync/_schema.yml

```diff
@@ -19,3 +19,22 @@ models:
             - evt_index
       - check_dex_base_trades_seed:
           seed_file: ref('maverick_zksync_base_trades_seed')
+
+  - name: pancakeswap_v2_zksync_base_trades
+    meta:
+      blockchain: zksync
+      sector: dex
+      project: pancakeswap
+      contributors: chef_seaweed, tomfutago, lgingerich
+    config:
+      tags: [ 'zksync', 'dex', 'trades', 'pancakeswap', 'uniswap', 'v2' ]
+    description: "Pancakeswap zkSync v2 base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      # - check_dex_base_trades_seed:
+      #     # seed_file: ref('pancakeswap_zksync_base_trades_seed')
+      #     filter:
+      #       version: 2
```

Comment:
I just added the sources.
You can continue adding the seed file and testing the spell.